<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecordDlg</class>
 <widget class="QDialog" name="AddRecordDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter record data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="surnameEdit"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="patronymicEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="regionLabel">
         <property name="text">
          <string>Region</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLineEdit" name="amountEdit"/>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="contactEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="patronymicLabel">
         <property name="text">
          <string>Patronymic</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="loanGuaranteeLabel">
         <property name="text">
          <string>Loan guarantee</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="regionComboBox"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="amountLabel">
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="contactLabel">
         <property name="text">
          <string>Contact</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="placeLabel">
         <property name="text">
          <string>Place</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QComboBox" name="belonginComboBox"/>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="activityEdit"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="activityLabel">
         <property name="text">
          <string>Activity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="surnameLabel">
         <property name="text">
          <string>Surname</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLineEdit" name="loanGuaranteeEdit"/>
       </item>
       <item row="11" column="2">
        <widget class="QComboBox" name="colorComboBox"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="clorLabel">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="belongingLabel">
         <property name="text">
          <string>Belonging</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="placeEdit"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="commentLabel">
         <property name="text">
          <string>Comment</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QLineEdit" name="commentLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="photoLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="text">
          <string>photo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="photoEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="loadPhotoButton">
         <property name="text">
          <string>Load Photo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removePhotoButton">
         <property name="text">
          <string>Remove Photo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>surnameEdit</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>patronymicEdit</tabstop>
  <tabstop>regionComboBox</tabstop>
  <tabstop>placeEdit</tabstop>
  <tabstop>contactEdit</tabstop>
  <tabstop>belonginComboBox</tabstop>
  <tabstop>activityEdit</tabstop>
  <tabstop>amountEdit</tabstop>
  <tabstop>loanGuaranteeEdit</tabstop>
  <tabstop>colorComboBox</tabstop>
  <tabstop>photoEdit</tabstop>
  <tabstop>loadPhotoButton</tabstop>
  <tabstop>removePhotoButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddRecordDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddRecordDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
